image_tools/README.md

```diff
@@ -0,0 +1,33 @@
+## **What Is This?**
+
+This demo provides a simple utility to **connect to a workstation default camera device** and **display** it in an **OpenCV** window like below:
+
+![](img/result.png)
+
+## **Build**
+
+```bash
+colcon build --package-select image_tools
+```
+
+## **Run**
+
+In `image_tools` ROS 2 package, 2 executables are provided, namely `cam2image` and `showimage` with different functions.
+
+## **1 - cam2image**
+Running this executable connects to your workstation's default camera device's video stream and publishes the images on '/image' and '/flipimage' topics using a ROS 2 publisher.
```

Comment:
I think it is worthwhile pointing out that this node has a number of options, particularly the one that allows you to produce "burgers" if you don't have a camera device.  If you do `ros2 run image_tools cam2image --help`, you'll see the examples.

Reply:
![image](https://user-images.githubusercontent.com/19281974/214325576-ee4d5595-04b4-4e01-a2db-1b12d596a0c2.png)

Interesting :exploding_head: 

Thank you for the tip. Will include it in the `README.md`.

Reply:
Done.